kubernetes-alternator-demo/cluster.yaml

```diff
@@ -0,0 +1,118 @@
+# Namespace where the Scylla Cluster will be created
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: scylla
+
+---
+
+# Role for scylla members.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scylla-cluster-member
+  namespace: scylla
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+    verbs:
+      - get
+  - apiGroups:
+      - ""
+    resources:
+      - services
+    verbs:
+      - get
+      - list
+      - patch
+      - watch
+  - apiGroups:
+      - "apps"
+    resources:
+      - statefulsets
+    verbs:
+      - get
+      - list
+      - patch
+      - watch
+  - apiGroups:
+      - scylla.scylladb.com
+    resources:
+      - clusters
+    verbs:
+      - get
+
+---
+
+# ServiceAccount for scylla members.
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: scylla-cluster-member
+  namespace: scylla
+
+---
+
+# RoleBinding for scylla members.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: scylla-cluster-member
+  namespace: scylla
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: scylla-cluster-member
+subjects:
+  - kind: ServiceAccount
+    name: scylla-cluster-member
+    namespace: scylla
+
+---
+
+# Scylla Cluster
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: Cluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: scylla-cluster
+  namespace: scylla
+spec:
+  version: 4.0.0
+  alternator:
+    port: 8000
+  agentVersion: 2.0.2
+  cpuset: true
+  sysctls:
+    - "fs.aio-max-nr=2097152"
+  datacenter:
+    name: <gcp_zone>
+    racks:
+      - name: <gcp_region>
+        scyllaConfig: "scylla-config"
+        scyllaAgentConfig: "scylla-agent-config"
+        members: 3
```

Comment:
We have 4 scylla k8s nodes  and we use only 3 out of 4.

Reply:
It just takes so long to boot them....